projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli_impl.py

```diff
@@ -14,7 +15,6 @@
 from taurus_datajob_api import DataJobContacts
 from taurus_datajob_api import DataJobDeployment
 from taurus_datajob_api import DataJobSchedule
```

Comment:
Unrelated but isn't taurus_datajob_api an internal vmware resource? Should we have it as a vdk-control-cli dependency?

Reply:
No, this is part of vdk-control-service-api  which is autogenerate from api.yaml open API spec. We have set the package name to be this: https://github.com/vmware/versatile-data-kit/blob/4cdbe3b0cfedb8ddc675bec27aee4a88389ce16d/projects/control-service/projects/model/apidefs/datajob-api/config-python.json#L2

We can change it to vdk_datajob_api probably as it makes more sense.

Reply:
Gotcha.